examples/basic-tracer-node/index.js

```diff
@@ -17,13 +17,13 @@ if (EXPORTER.toLowerCase().startsWith('z')) {
   exporter = new JaegerExporter(options);
 }
 
-const tracer = new BasicTracer();
+const registry = new BasicTracerRegistry();
```

Comment:
IMO this (initialization of `BasicTracerRegistry` / `NodeTracerRegistry`) seems a little burdensome (or extra step) for the end users. I was thinking something like this:
```js
// In case of BasicTracer
const opentelemetry = require('@opentelemetry/tracing');
opentelemetry.getTracerRegistry().getTracer(); // should return the default BasicTracer.
opentelemetry.getTracerRegistry().getTracer('mongodb'); // with name
opentelemetry.getTracerRegistry().getTracer('redis', '1.0.0'); // with name and version
opentelemetry.getTracerRegistry().addSpanProcessor(...);

// In case of NodeTracer
const opentelemetry = require('@opentelemetry/node');
opentelemetry.getTracerRegistry().getTracer(); // should return the default NodeTracer.

// In case of WebTracer
const opentelemetry = require('@opentelemetry/web');
opentelemetry.getTracerRegistry().getTracer(); // should return the default WebTracer.
```

The `getTracerRegistry` function is responsible for returning the corresponding `TracerRegistry` and we should create (and assign) the `TracerRegistry` instance statically once you load the library (maybe in `index.ts`). WDYT?

Reply:
I like this idea a lot. Actually in this instance you would be able to just do `opentelemetry.getTracer` directly because we can assume you are using the default tracer factory

Reply:
That would be very cool, but looks like `getTracerRegistry` is required in order to add the `SpanProcessor` for all the tracers.

Reply:
I was thinking about this and if you call `getTracerRegistry` before a tracer registry is initialized you would receive the `NoopTracerRegistry`. This is the same as if you called `getTracer` before without initializing a global tracer.

Reply:
In your case, if you had `core` and `node` both loaded, they would have different global tracers so this would probably be confusing. I think the current way it works is ok and I don't want to overcomplicate the PR with changes like that for now.